### DISCUSSION TITLE
Choose Arrow type

### DISCUSSION THREAD

**Discussion by Shigeto-Amatake** [excalidraw/excalidraw#8370] [Q&A]
Is the arrow type, which can be selected in the demo app, not yet implemented?
Or do I need to set some options?

I love elbow arrow.
Thank you.

![image](https://github.com/user-attachments/assets/4435ac77-fa9c-4fe7-b4d3-e46598f3fc31)

**Comment by Mrazator** [comment]
Elbow arrows are shipped!
